.env.template.yml

```
@@ -0,0 +1,44 @@
+# -----------------------------------------------------------------------------
+# INSTRUCTIONS:
+# 1. Rename this file to `.env` in the root of the project.
+# 2. Replace all placeholder values (e.g., "changeme") with actual secrets/configs.
+#
+# NOTE: This file contains all necessary "keys" for the Phase 0 stabilization effort.
+# -----------------------------------------------------------------------------
+
+# --- General Compose & Docker Settings ---
+# Standardize platform to avoid issues on ARM-based machines (e.g., Apple M1/M2).
+DOCKER_DEFAULT_PLATFORM=linux/amd64
+COMPOSE_PROJECT_NAME=ovs_stabilization
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The COMPOSE_PROJECT_NAME key should go before the DOCKER_DEFAULT_PLATFORM key

(UnorderedKey)